webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVReassignHostsForm.js

```diff
@@ -122,12 +153,24 @@ const CVReassignHostsForm = () => {
           <p>{multiCVRemovalInfo}</p>
         </Alert>
       )}
+      {showAlert &&
+        <Alert
+          variant="info"
+          ouiaId="disabled-environments-alert"
+          isInline
+          title={__('Some environments are disabled because they are not associated with the host\'s content source.')}
+          style={{ marginBottom: '1rem' }}
+        >
+          {__('To enable them, add the environment to the host’s content source.')}
```

Comment:
```suggestion
          {__('To enable them, add the environment to the hosts\' content sources. Or, assign a new content source to the hosts.)}
```

And the "assign.." part can be a link like we were talking about.